Mathlib/Algebra/Central/Basic.lean

```diff
@@ -54,3 +55,16 @@ lemma baseField_essentially_unique
   exact ⟨x', (algebraMap K D).injective <| by simp [← H, algebraMap_eq_smul_one]⟩
 
 end Algebra.IsCentral
+
+section MatrixAlgebra
+
+variable (K : Type u) [Field K]
+
+open Matrix in
+/-- Matrix Algebra is central -/
```

Comment:
This docstring is in some sense worse than no docstring, you can get more information by just reading the statement of the theorem.

Reply:
hmmm okay deleted